### PR TITLE
customize keltner channel ATRs

### PR DESCRIPTION
**Pull request by ludydoo**
Allow users to define custom number of ATRs away from the central moving average

https://school.stockcharts.com/doku.php?id=technical_indicators:keltner_channels

### CODE REVIEW

**Comment by xarthurx** [comment]
Hi, this is actually the multiplier, not the parameter for ATR.
The parameter `window_atr` is already for changing the ATR range.

**Comment by xarthurx** [comment]
The pull request same as the other one: https://github.com/bukosabino/ta/pull/259

File: ta/volatility.py [hunk 1]
```diff
@@ -247,8 +250,8 @@ def _run(self):
                 window=self._window_atr,
                 fillna=self._fillna,
             ).average_true_range()
-            self._tp_high = self._tp + (2 * atr)
-            self._tp_low = self._tp - (2 * atr)
+            self._tp_high = self._tp + (n_atr * atr)
```
Comment (bukosabino) [inline ta/volatility.py:253]: Should be `self.n_atr`

File: ta/volatility.py [hunk 2]
```diff
@@ -247,8 +250,8 @@ def _run(self):
...
+            self._tp_low = self._tp - (n_atr * atr)
```
Comment (bukosabino) [inline ta/volatility.py:254]: Should be `self.n_atr`

**Review by bukosabino — COMMENTED** [review]
Hi @ludydoo 

Looks good to me, please, fix the code and we can merge!

**Comment by bukosabino** [comment]
Finally, I have merged this one: https://github.com/bukosabino/ta/pull/259

Thanks, @ludydoo @xarthurx

**Comment by bukosabino** [comment]
Included this feature on the last version of the library -> v0.10.1

**Pull request closed without merging**